common/src/main/java/io/github/axolotlclient/modules/hud/gui0/hud/simple/PingHud.java

```diff
@@ -75,12 +60,12 @@ public boolean tickable() {
 	@Override
 	public void tick() {
 		if (second >= refreshDelay.get() * 20) {
-			updatePing();
+			//updatePing();
 			second = 0;
 		} else
 			second++;
 	}
-
+/*
```

Comment:
This should probably be implemented by just delegating to a method implemented in the bridge, since this ping code differs slightly across versions it really doesn't work in the common module.